src/library/Library.java

```diff
@@ -75,4 +78,41 @@ public int getLibrarySize() {
     public Book getBookByIndex(int i) {
         return books.get(i);
     }
+
+    public List<Book> getSortedBooks(String sortingOrder, String sortingType) {
```

Comment:
yeah ))) almost cool.
You can move sorting orders to enum with values 1 and 0 for example. And use it instead of comparing Strings. But this way it will works too.